SPECS/glibmm/glibmm.spec

```diff
@@ -0,0 +1,159 @@
+%global apiver 2.68
+
+Summary:        C++ interface to the glib
+Name:           glibmm
+%define BaseVersion 2.70
+Version:        %{BaseVersion}.0
+Release:        1%{?dist}
+License:        LGPLv2+
+URL:            https://developer.gnome.org/glibmm/stable/
+Group:          Applications/System
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        http://ftp.gnome.org/pub/GNOME/sources/glibmm/%{BaseVersion}/glibmm-%{version}.tar.xz
+BuildRequires:  doxygen
+BuildRequires:  gcc-c++
+BuildRequires:  glib-devel 
+BuildRequires:  glib-schemas
+BuildRequires:  graphviz
+BuildRequires:  libsigc++30
+BuildRequires:  libxslt
+BuildRequires:  m4
+BuildRequires:  meson
+BuildRequires:  mm-common
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+
+%global with_check 0
```

Comment:
Please never hard-code the value of `with_check`. We set it automatically depending on the build we're currently running.
```suggestion
```